x/dex/keeper/msgserver/msg_server_register_pairs.go

```diff
@@ -0,0 +1,25 @@
+package msgserver
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/sei-protocol/sei-chain/x/dex/types"
+)
+
+func (k msgServer) RegisterPairs(goCtx context.Context, msg *types.MsgRegisterPairs) (*types.MsgRegisterPairsResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	// TODO: add validation such that only the user who stored the code can register pairs
```

Comment:
can we add basic msg validity check (like pair != empty, creator exists and etc), and unit tests for those checks

Reply:
Added as part of `ValidateBasic`